src/GraphQl/Resolver/ResourceAccessCheckerTrait.php

```diff
@@ -11,15 +11,6 @@
 
 declare(strict_types=1);
 
-/*
- * This file is part of the API Platform project.
- *
- * (c) Kévin Dunglas <dunglas@gmail.com>
```

Comment:
Move up the header license message than `declare(strict_types=1);`

Reply:
It was duplicated. The copyright header was already there. :laughing: